playbooks/roles/elasticsearch/defaults/main.yml

```diff
@@ -15,3 +15,5 @@ elasticsearch_group: "elasticsearch"
 ELASTICSEARCH_CLUSTER_MEMBERS: []
 ELASTICSEARCH_HEAP_SIZE: "512m"
 ELASTICSEARCH_VERSION: "0.90.13"
+ELASTICSEARCH_HOST: localhost
```

Comment:
The `elasticsearch` role itself uses the above `ELASTICSEARCH_CLUSTER_MEMBERS` to specify hosts internally. To reference the host (or cluster of hosts) in another role, let's crib from the `forum` role [1], so both the `kibana` and the `logstash` roles would need
```yaml
KIBANA_ELASTICSEARCH_HOST: localhost
KIBANA_ELASTICSEARCH_PORT: 9200
```

and

```yaml
LOGSTASH_ELASTICSEARCH_HOST: localhost
LOGSTASH_ELASTICSEARCH_PORT: 9200
```

respectively.

Roles should generally avoid depending on variables from other roles, `common` being a notable exception. This is particularly important because those variables in `defaults/main.yml` are only set iff you include that role in that playbook. While we are installing both e + l/k now, we shouldn't assume that will always be the case. Say, we wanted to update just logstash but not elasticsearch; we can only do that if logstash gets its own copy of the variable.

[1] https://github.com/Stanford-Online/configuration/blob/5ff7a66661bc4e7f68c759d4975536af6ca7f4a3/playbooks/roles/forum/defaults/main.yml#L27-L29